src/components/BreakdownRow/index.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles/index";
+
+const useStyles = makeStyles({
+  row: {
+    minHeight: 40,
+    fontSize: 14,
+    borderBottom: "1px solid #0000001f",
+  },
+  box: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    textAlign: "center",
+    flex: 1,
+    paddingBottom: 2,
+    paddingTop: 2,
+  },
+  grey: {
+    backgroundColor: "#0000001f",
+  },
+  lightRed: {
+    backgroundColor: "#ff000011",
+  },
+  red: {
+    backgroundColor: "#ff000022",
+  },
+  lightBlue: {
+    backgroundColor: "#0000ff11",
+  },
+  blue: {
+    backgroundColor: "#0000ff22",
+  },
+  total: {
+    fontWeight: "bold",
+  },
+});
+
+const BreakdownRow = ({
+  data,
```

Comment:
Is there a reason you're passing `data` in as an array, as opposed to breaking up the label and red/blue values to be more verbose about it?